addons/spreadsheet_dashboard_hr_expense/data/files/expense_sample_dashboard.json

```diff
@@ -0,0 +1,808 @@
+                            "color": "#434343"
+                        },
+                        "type": "scorecard",
+                        "background": "#FFF7ED",
+                        "keyValue": "Data!C4",
+                        "humanize": false
+                    }
+                },
+                {
+                    "id": "3ce1b146-06b2-4a52-8a21-183d9b25af67",
+                    "x": 0,
+                    "y": 178,
+                    "width": 1002,
+                    "height": 348,
+                    "tag": "chart",
+                    "data": {
+                        "type": "bar",
+                        "dataSetsHaveTitle": true,
+                        "dataSets": [
+                            {
+                                "dataRange": "Data!B7:B14",
+                                "yAxisId": "y"
+                            },
+                            {
+                                "dataRange": "Data!C7:C14"
+                            },
+                            {
+                                "dataRange": "Data!D7:D14"
+                            },
+                            {
+                                "dataRange": "Data!E7:E14"
+                            }
+                        ],
+                        "legendPosition": "top",
+                        "labelRange": "Data!A7:A14",
+                        "title": {},
+                        "stacked": true,
+                        "aggregated": false,
+                        "horizontal": false
+                    }
+                }
+            ],
+            "tables": [],
+            "areGridLinesVisible": true,
+            "isVisible": true,
+            "headerGroups": {
+                "ROW": [],
+                "COL": []
+            },
+            "dataValidationRules": [],
+            "comments": {}
+        },
+        {
+            "id": "44bfb206-c15d-4314-a394-97690f1d5e8e",
+            "name": "Data",
+            "colNumber": 26,
+            "rowNumber": 98,
+            "rows": {},
+            "cols": {
+                "0": {
+                    "size": 117
+                },
+                "1": {
+                    "size": 94
+                },
+                "2": {
+                    "size": 93
+                },
+                "3": {
+                    "size": 147
+                },
+                "4": {
+                    "size": 100
+                },
+                "5": {
+                    "size": 192
+                },
+                "6": {
+                    "size": 175
+                },
+                "7": {
+                    "size": 146
+                },
+                "8": {
+                    "size": 53
+                },
+                "9": {
+                    "size": 105
+                },
+                "10": {
+                    "size": 65
+                }
+            },
```

Comment:
no need

Reply:
The sizing is for the tables display 🙂 
<img width="502" alt="image" src="https://github.com/user-attachments/assets/b669cfe9-2d2b-4a63-9a7e-97cc711eff94">

Reply:
The sizing here are on the sheet `Data`, so not needed

Reply:
ah I see 👍